### DISCUSSION TITLE
Agregar más opciones para el PBTaskAddEditor (visualizador / seguidor) o crear un PB que tenga esas opciones

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#571] [3. Ideas & Feature Requests] [answered]
Hola a todos

Me gustaría ver la posibilidad de que se agregue, como campo, los términos: visualizador y seguidor en el bot pbtaskaddeditor o en otro pbbot, para que en momento de configurar una rutina, se pueda distribuir los usuarios que se agreguen al canal sin que todos sean editores. Lo pregunto porque el pbupdatetask puede pisar usuarios existentes, reemplazando por los nuevos que se agreguen.

Está la opción de usar la vista global, para que todos puedan ver la tasks . El problema con eso, es que si quiero invitar al boss del usuario como visualizador, no debiera ver las otras tasks que no sean del súbdito.

Gracias =>

**Comment by acedward** [comment, marked as answer]
Una forma de hacerlo es usar efectivamente PBUpdateTask
https://doc.cotalker.com/docs/documentation/automation/bots/pbupdatetask-2.0.1

Y cuando se actualiza la tarea se pueden pasar los valores actuales más los nuevos:

![image](https://user-images.githubusercontent.com/3951895/147006597-6b230929-c3ad-4a9f-a7de-7dabd6e38924.png)

**Reply by NicolasPLG** [reply]
hola @acedward  muchas gracias por la información lo probaré. cerraré el tickete
